tests/scripts/test-ref-configs.pl

```diff
@@ -32,15 +32,27 @@
         'compat' => '-m tls12 -f \'^TLS-PSK-WITH-AES-...-CCM-8\'',
         'test_again_with_use_psa' => 1
     },
+    'config-ccm-psk-dtls1_2.h' => {
+        'compat' => '-m dtls12 -f \'^TLS-PSK-WITH-AES-...-CCM-8\'',
```

Comment:
It would make sense to test TLS 1.2 as well as DTLS 1.2. But that doesn't work: `m->m` tests pass but `m->G` tests fail. With debug enabled:
```
ssl_msg.c:3620: |3| input record: msgtype = 23, version = [3:3], msglen = 821
ssl_msg.c:1871: |2| => fetch input
ssl_msg.c:1882: |1| requesting more data than fits
ssl_msg.c:4653: |1| mbedtls_ssl_fetch_input() returned -28928 (-0x7100)
ssl_msg.c:3917: |1| ssl_get_next_record() returned -28928 (-0x7100)
ssl_msg.c:5394: |1| mbedtls_ssl_read_record() returned -28928 (-0x7100)
 mbedtls_ssl_read returned -0x7100
```
If I understand correctly, GnuTLS is trying to send a larger record than what our build supports. This works with DTLS thanks to fragmentation support, but not with TLS.

I'm not sure if this should be filed as a known defect in `compat.sh`.